algo_demo/style.css

```
@@ -1,124 +1,401 @@
+
+  --viz-cat-1: #ffffff; --viz-cat-2: #3fb950; --viz-cat-3: #d29922;
+  --viz-cat-4: #db61a2; --viz-cat-5: #39c5cf; --viz-cat-6: #a371f7;
+  --viz-cat-7: #f85149; --viz-cat-8: #58a6ff;
+}
+
+::selection { background: var(--selection-bg); color: var(--selection-color); }
+
+/* ── Typography ─── */
+body {
+  font-family: 'SF Mono', 'JetBrains Mono', 'IBM Plex Mono', ui-monospace,
+               'Cascadia Mono', 'Fira Code', 'Menlo', 'Courier New', monospace;
```

Stylelint (17.14.0)

[error] Expected no quotes around "Menlo" (font-family-name-quotes)

(font-family-name-quotes)